query.go

```diff
@@ -1077,7 +1078,11 @@ func (s *ChainService) sendTransaction(tx *wire.MsgTx, options ...QueryOption) e
 							tx, nil, qo.encoding,
 						)
 
-						numReplied++
+						// Peers might send the INV
```

Comment:
In practice, won't the closing of `peerQuit` below mean that for each peer this'll only ever be incremented once?

Reply:
I think so, yes. That's why I removed the counter and replaced it with a map that just tracks if we got _any_ reply or not.